engine/storage/integration-test/test/org/apache/cloudstack/storage/test/FakePrimaryDataStoreDriver.java

```diff
@@ -65,8 +66,13 @@ public long getUsedIops(StoragePool storagePool) {
     }
 
     @Override
-    public long getVolumeSizeIncludingHypervisorSnapshotReserve(Volume volume, StoragePool pool) {
-        return volume.getSize();
+    public long getDataObjectSizeIncludingHypervisorSnapshotReserve(DataObject dataObject, StoragePool pool) {
+        return dataObject.getSize();
+    }
+
+    @Override
+    public long getBytesRequiredForTemplate(TemplateInfo templateInfo, StoragePool storagePool) {
+        return 0;
```

Comment:
Minor Nit: ``0`` is an integer value.  ``0L`` is a long value.  Java automatically widens the value, but it is more hygienic to match the type for a constant value.